flutter-toc.html

```diff
@@ -98,15 +98,15 @@
 	<li><a href="/Flutter/right-to-left">Right to Left (RTL)</a></li>
     <li><a href="/Flutter/themes">Themes</a></li>
     <li>
-		AssistView
+		AI AssistView
 		<ul>
-			<li><a href="/Flutter/assistview/overview">Overview</a></li>
-      <li><a href="/Flutter/assistview/getting-started">Getting Started</a></li>
```

Comment:
In link - correct the widget name with casing

Reply:
Hi @VijayakumarMariappan sir,

I have modified the folder name AIAssistView to ai-assistview in the toc file, checked and ensured in all places of UG docs.

Regards,
Thankyou